Automattic/jetpack
# E2E tests: Move Search tests into Search plugin folder

#### Changes proposed in this Pull Request:

Moved search end-to-end tests into a standalone test package.

#### Jetpack product discussion
pd5faL-3G-p2

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
- Check CI:
	- All tests should pass
	- Search tests should run in a separate job
	- Jetpack job should not contain any search tests
	- See https://github.com/Automattic/jetpack/actions/runs/2004371534
